.github/workflows/build.yml

```diff
@@ -29,7 +32,9 @@ jobs:
           printf '%*s\n' "$COLUMNS" '' | tr ' ' -
           echo "Testing game: $game"
           pushd "$game" > /dev/null
-          npm install
+          npm uninstall engine
+          npm install engine@file:../../engine/jellyfish.js-0.0.2.tgz
```

Comment:
Can you not hardcode the version number? Something like `npm install "engine@file:$(printf '%s\n' ../../engine/jellyfish.js-*.tgz | sort -Vr | head -n1)"` perhaps?